mailer.js

```diff
@@ -26,6 +27,19 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocalTimeString (timeZone, locale) {
+    // if no timeZone is passed, use 'Etc/UTC'
+    moment.tz.setDefault('Etc/UTC')
+    // if no locale is passed, use 'en'
+    locale = locale || 'en'
+    moment.updateLocale(locale)
```

Comment:
@shane-tomlinson From same doc, http://momentjs.com/docs/#/customization/ , it says `2.12.0 deprecated using moment.locale() to change an existing locale. Use moment.updateLocale() instead.`
Should I go ahead and use `moment.locale`?